integration/pom.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>net.adoptopenjdk</groupId>
+        <artifactId>icedtea-web-parent</artifactId>
+        <version>1.0-SNAPSHOT</version>
+        <relativePath>../</relativePath>
+    </parent>
+
+    <artifactId>icedtea-web-integration-tests</artifactId>
```

Comment:
please provide properties that describe intention of the module, such as:
    <name>Integration Tests</name>
    <description>IcedTea-Web integration tests for the main JNLP application execution scenarios.</description>

Reply:
Done